package.json

```diff
@@ -9,7 +9,7 @@
     "flow-typed": "rm -rf flow-typed && flow-typed install",
     "flow": "flow",
     "karma": "cross-env NODE_ENV=test babel-node $(npm bin)/karma start",
-    "test": "if [ ! $SKIP_TEST ]; then npm run lint && npm run flow-typed && npm run flow && npm run jest-ssr && npm run karma && npm run jest-screenshot; fi;",
+    "test": "if [ ! $SKIP_TEST ]; then npm run lint && npm run flow-typed  && npm run karma && npm run jest-screenshot; fi;",
```

Comment:
Please back this out. Why remove flow? We can't guarantee quality without good static typing.

Reply:
done